panel/panelpluginsmodel.cpp

```diff
@@ -284,7 +284,9 @@ QPointer<Plugin> PanelPluginsModel::loadPlugin(LXQt::PluginInfo const & desktopF
 
 QString PanelPluginsModel::findNewPluginSettingsGroup(const QString &pluginType) const
 {
-    QStringList groups = mPanel->settings()->childGroups();
+    QString userConfigDir = QStandardPaths::writableLocation(QStandardPaths::ConfigLocation);
+    QSettings userSettings(QStringLiteral("%1/lxqt/panel.conf").arg(userConfigDir), QSettings::IniFormat);
```

Comment:
There isn't a reason that the config file should be `panel.conf`, to say nothing of its path. The panel config file I'm using right now is `panel-kwin_wayland.conf`.

https://github.com/lxqt/lxqt-panel/issues/2126 was rather a question, whose answer you found. It was by no means a bug, and can be closed.

Reply:
I'll close this PR because I'm not very familiar with Qt's system integration, nor LXQt settings.

I think it is a bug to have "<pluginname>2" as the first numbering, due to system config, though.

Reply:
> I think it is a bug…

By definition, it isn't.